src/common/settings_objects.h

```diff
@@ -166,7 +166,11 @@ namespace PowerToysSettings {
         return output.data();
       }
     }
-    HotkeyObject(web::json::value hotkey_json) : m_json(hotkey_json) {};
+    HotkeyObject(web::json::value hotkey_json) : m_json(hotkey_json) {
+      if (m_json.has_number_field(L"code")) {
+        m_json.as_object()[L"key"] = web::json::value::string(key_from_code(get_code()));
+      }
+    };
```

Comment:
For existing users the new feature will never work?

Reply:
Or will a save remove the "code" field?

Reply:
This is for users with saved fancyzones setting.

Reply:
But if they change any setting and save, the "code" field will be removed?
I want to understand if they will ever see the new key mapping.

Reply:
It overwrites the "key" field, which stores the value displayed in the hotkey input field.